### PR TITLE
[DPEDE-1416] Add pending functional tests to date picker web component

### PR DESCRIPTION
**Pull request by xft-dev**
[https://ctl.atlassian.net/browse/DPEDE-1416](https://ctl.atlassian.net/browse/DPEDE-1416)

Added 3 new tests for minutesStep attribute in Date Picker with values 10, 20 and 1 in order to test all possible cases.

### CODE REVIEW

**Review by josecarlosgonzalezv — APPROVED** [review]

File: cypress/e2e/chi-custom-elements/date-picker.cy.js [hunk 1]
```diff
@@ -605,7 +608,145 @@ describe('Date picker', function() {
...
+    cy.get('@testTimeMinutesStep')
+      .find('input')
+      .scrollIntoView()
+      .focus()
+      .get('@testTimeMinutesStep')
+      .find('chi-popover[active]')
+      .should('have.attr', 'active');
+
+      cy.get('@testTimeMinutesStepPicker')
+      .should('have.attr', 'minutes-step', `${minutesStep4}` );
```
Comment (josecarlosgonzalezv) [inline cypress/e2e/chi-custom-elements/date-picker.cy.js:643]:
```suggestion
    cy.get('@testTimeMinutesStepPicker')
      .should('have.attr', 'minutes-step', `${minutesStep4}` );
```

File: cypress/e2e/chi-custom-elements/date-picker.cy.js [hunk 2]
```diff
@@ -605,7 +608,145 @@ describe('Date picker', function() {
...
+      .find(`.${TIME_PICKER_MINUTE}`)
+      .contains(noonHour)
+      .should('have.class', '-active');
+
+    // Iterate through minutes-list to verify its length
+    cy.get('@testTimeMinutesStep')
+      .find(`.${TIME_PICKER_MINUTE}`)
+      .each((item, index, list) => {
+        //cy.log( item.text() );
+      })
```
Comment (josecarlosgonzalezv) [inline cypress/e2e/chi-custom-elements/date-picker.cy.js:655]:
```suggestion
      .each((item, index, list) => {
        //cy.log( item.text() );
      })
```

Needed?

File: cypress/e2e/chi-custom-elements/date-picker.cy.js [hunk 3]
```diff
@@ -15,7 +15,10 @@ const minute = '30';
 const period = 'PM';
 const minutesStep = '5';
 const minutesStep2 = '30';
-const minutesStepDefault = '15';
+const minutesStep3 = '15';
+const minutesStep4 = '10';
+const minutesStep5 = '20';
+const minutesStep6 = '1';
```
Comment (josecarlosgonzalezv) [inline cypress/e2e/chi-custom-elements/date-picker.cy.js:21]:
```suggestion
const minutesStep1 = '1';
const minutesStep10 = '10';
const minutesStep15 = '15';
const minutesStep20 = '20';
```

**Review by josecarlosgonzalezv — CHANGES_REQUESTED** [review]
Tests are repeated, can we find some way with functions to reduce these tests?

File: cypress/e2e/chi-custom-elements/common/data-picker-common.cy.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
+export const DATE_PICKER_MINUTESSTEP_VALUES = [
```
Comment (ASanchLumen) [inline cypress/e2e/chi-custom-elements/common/data-picker-common.cy.js:1]: There's a typo in the filename! datE-...  😸

Comment (xft-dev) [reply cypress/e2e/chi-custom-elements/common/data-picker-common.cy.js:1]: Ok, I change it in the next commit. Thanks. 

Also, there will be more changes in order to improve the code.

**Review by josecarlosgonzalezv — APPROVED** [review]

**Review by ASanchLumen — APPROVED** [review]

**Review by jllr — APPROVED** [review]

**Pull request merged**